main.js

```diff
@@ -353,7 +367,7 @@ function car_collision() {
 
 function water_collision() {
     for (var i = 15; i < 30; i = i+3) {
-        var obs = obstacleArray[i];
+        var obs = obstacleArray[i];- obs.dw/2
```

Comment:
I think this might be a typo. Please confirm. Should it have been obstacleArray[i] - obs.dw/2; ?

Reply:
Yeah thats what I checked as well, it appears to be moved from 398

Reply:
Fixed that typo and pushed back to var-organization branch